### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character '\u2026' in position 24: ordinal not in range(128)

### ISSUE THREAD

**Issue by ignatenkobrain** [mesonbuild/meson#1436]
```
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/mesonbuild/mesonmain.py", line 286, in run
    app.generate()
  File "/usr/lib/python3.5/site-packages/mesonbuild/mesonmain.py", line 168, in generate
    intr.run()
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreter.py", line 2455, in run
    super().run()
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreterbase.py", line 125, in run
    self.evaluate_codeblock(self.ast, start=1)
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreterbase.py", line 146, in evaluate_codeblock
    raise e
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreterbase.py", line 140, in evaluate_codeblock
    self.evaluate_statement(cur)
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreterbase.py", line 151, in evaluate_statement
    return self.function_call(cur)
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreterbase.py", line 372, in function_call
    return self.funcs[func_name](node, self.flatten(posargs), kwargs)
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreterbase.py", line 47, in wrapped
    return f(self, node, args, kwargs)
  File "/usr/lib/python3.5/site-packages/mesonbuild/interpreter.py", line 1643, in func_message
    mlog.log(mlog.bold('Message:'), argstr)
  File "/usr/lib/python3.5/site-packages/mesonbuild/mlog.py", line 82, in log
    print(*arr, file=log_file, **kwargs) # Log file never gets ANSI codes.
UnicodeEncodeError: 'ascii' codec can't encode character '\u2026' in position 24: ordinal not in range(128)
WARNING: You are using 'ANSI_X3.4-1968' which is not a a Unicode-compatible locale.
WARNING: You might see errors if you use UTF-8 strings as filenames, as strings, or as file contents.
WARNING: Please switch to a UTF-8 locale for your platform.
The Meson build system
Version: 0.39.0.dev1
```

**Comment by ignatenkobrain** [comment]
I suppose we need to open `log_file` with `codecs.open()` in utf-8 mode

**Comment by ignatenkobrain** [comment]
Ohh, fixed in #1432
